src/index.js

```diff
@@ -1196,6 +1223,8 @@ BigQuery.prototype.job = function(id) {
  * @param {string|object} query A string SQL query or configuration object.
  *     For all available options, see
  *     [Jobs: query request body](https://cloud.google.com/bigquery/docs/reference/v2/jobs/query#request-body).
+ * @param {string} [query.location] The geographic location of the job.
```

Comment:
Should `location` be an argument to the BigQuery constructor as well, that would be used for this operation, and descend to the other levels?

Reply:
sure

Reply:
Done.